File: Python/sign.py
#This program tells if the given number is Positive, Negative or Zero.

n = int(input("Enter a number :\n"))

if(n==0):
    print("The number is a Zero")
elif(n>0):
    print(f"The number {n} is Positive")
else:
    print(f"The number {n} is Negative")

File: C/sign.c
//This program tells if the given number is Positive, Negative or Zero.

#include<stdio.h>
void check(int n);

void main(){
    int n;
    printf("Enter a number :\n");
    scanf("%d",&n);
    check(n);
}

void check(int n){

    if(n==0){
        printf("The number is a Zero");
    }
    else if(n>0){
        printf("The number %d is Positive",n);
    }
    else{
        printf("The number %d is Negative",n);
    }
}